### PR TITLE
Link to ConfigFile in Saving Gamess tutorial

### PR DESCRIPTION
**Pull request by nikhilCad**
I think that ConfigFile provides a much simpler solution to saving data. Hence, the user should be introduced to it in this tutorial.

### CODE REVIEW

File: tutorials/io/saving_games.rst [hunk 1]
```diff
@@ -12,6 +12,8 @@ Advanced save game systems should allow for additional information about
 an arbitrary number of objects. This will allow the save function to
 scale as the game grows more complex.
 
+For simply saving user settings,you can refer to :ref:`class_configfile`.
```
Comment (Calinou) [inline tutorials/io/saving_games.rst]:
```suggestion
.. note::

    If you're looking to save user configuration, you can use the
    :ref:`class_configfile` class for this purpose.
```

**Comment by nikhilCad** [comment]
@Calinou can you please close this PR and add these changes youraelf. I currently can not access my PC due to some hardware failure and I am commenting this from a mobile.

**Comment by Calinou** [comment]
Looks good to me. Please squash all commits into a single commit, see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for instructions :slightly_smiling_face:

**Comment by nikhilCad** [comment]
I am currently out from home and hence can not access my PC. Can you possibly close this PR and make the changes manually?

**Comment by nikhilCad** [comment]
How much time does readthedocs take to update the pages. Its been over a week and the changes are not visible on the Saving games page

**Comment by Calinou** [comment]
@nikhilCad The change should already be visible on the `latest` branch, but it will only be visible in the `3.1` branch once the commit has been cherry-picked (which is done manually).

You don't have anything specific to do, we'll handle it :slightly_smiling_face:

**Pull request closed without merging**